libs/interfaces/FileDetails.ts

```diff
@@ -1,6 +1,46 @@
 import { FileType } from "./FileType";
 import { Item } from "./Item";
 
+export interface EmbeddedMetadataValues {
+  [key: string]:
+    | string
+    | number
+    | boolean
+    | Date
+    | Array<string | number | boolean | Date>
+}
+
+export interface AITagItem {
+  name: string
+  confidence: number
+  source: 'google-auto-tagging' | 'aws-auto-tagging'
+}
+
+export interface CmValues {
```

Comment:
@ahnv how do we send date?

Reply:
ISO String

Request
![image](https://github.com/imagekit-developer/imagekit-nodejs/assets/8640877/7298ce29-7f2c-4536-af11-898df5404030)

Response
![image](https://github.com/imagekit-developer/imagekit-nodejs/assets/8640877/108d9a39-fbcd-4f52-b638-eed21c30aab1)